addons/stock/migrations/11.0.1.1/post-migration.py

```diff
@@ -439,6 +439,12 @@ def create_stock_move_line_from_inventory_moves(env):
     )
 
 
+def fill_config_parameter_use_propagation_minimum_delta(env):
+    # new parameter, we set to true to maintain same behavior of v10
```

Comment:
Why don't you put this as docstring?

Reply:
Why should I? I don't care, but I will do if you care haha

Reply:
Each tool is for each usage. If you are documenting the aim of the method, use docstring. If it's a clarification about specific line/s of code, the use comments.

Reply:
Ok, done.